crates/iota/src/unit_tests/upgrade_compatibility_tests.rs

```diff
@@ -59,10 +57,18 @@ fn get_packages(name: &str) -> (Vec<CompiledModule>, Vec<CompiledModule>) {
     path.push("src/unit_tests/fixtures/upgrade_errors/");
     path.push(format!("{}_v2", name));
 
-    let pkg_v2 = BuildConfig::new_for_testing()
-        .build(&path)
-        .unwrap()
-        .into_modules();
+    let pkg_v2 = BuildConfig::new_for_testing().build(&path).unwrap();
+
+    (mods_v1, pkg_v2)
+}
 
-    (pkg_v1, pkg_v2)
+/// Snapshots will differ on each machine, normalize to prevent test failures
+fn normalize_path(err_string: String) -> String {
+    // test
+    let re = regex::Regex::new(r"^  ┌─ .*(\/fixtures\/.*\.move:\d+:\d+)$").unwrap();
+    err_string
+        .lines()
+        .map(|line| re.replace(line, "  ┌─ $1").into_owned())
+        .collect::<Vec<String>>()
+        .join("\n")
 }
```

Comment:
Hmmmmmmmmmmm what is this? Do we need this?

Reply:
It's used on the error in test_declarations_missing() and the error contains your the full path I guess, therefore on different machines it's a different error